scripts/ovpsim_log_to_trace_csv.py

```diff
@@ -340,15 +352,20 @@ def process_ovpsim_sim_log(ovpsim_log, csv, full_trace = 1, stop = 0,
             if len(split) == 1: continue
             item = split[1]
             if "----" in item: continue
-            if "REPORT" in line or item in [
+            if "REPORT" in line or item in [ # TODO sort csrs
                     "mtvec","pmpaddr0","pmpcfg0","mstatus","mepc","mscratch",
-                        "mcause","mtval","vl","vtype"]:
+                        "mcause","mtval","vl","vtype","sstatus"]:
+                logging.debug("Ignoring: [%d]  [[%s]]" % (instr_cnt, line))
                 pass
+            elif "Warning (RISCV_" in line:
+                logging.debug("Skipping: [%d] (%s) [[%s]]" % 
+                    (instr_cnt, prev_trace.instr_str, line))
+                prev_trace.instr = "nop"
+                prev_trace.instr_str = "nop"
             else:
-                pass
-                #fatal ("<unknown> (%s) in line: [%s] %s " %
-                #      (item, str(instr_cnt), line))
-  logging.info("Processed instruction count : %d" % instr_cnt)
+                fatal ("<unknown> (%s) in line: [%s] %s " %
```

Comment:
Do you want to have "if stop_on_first_error:" here?

Reply:
Tao
I always run my full regressions before submitting - so we have a probem in our environemtns being different - or some files wrong.
when I run

> python3 ./run.py -o ./outdir --simulator questa32 --simulator_yaml ../riscv-dv.overlay/yaml/simulator.yaml  --iss spike,ovpsim --test riscv_arithmetic_basic_test --iterations 1

it works
when I run with --target rv64gc I get an error... which is not stopping the run - and then fails to compile as no mabi and march set.

> python3 ./run.py -o ./outdir --simulator questa32 --simulator_yaml ../riscv-dv.overlay/yaml/simulator.yaml  --iss spike,ovpsim --test riscv_arithmetic_basic_test --iterations 1 --target rv64gc

**
> Unsupported pre-defined target: rv64gc
**

> Fri, 08 Nov 2019 21:52:59 INFO     Processing regression test list : /home/simond/git/tmp10/riscv-dv/target/rv64gc/testlist.yaml, test: riscv_arithmetic_basic_test
> Fri, 08 Nov 2019 21:52:59 INFO     Found matched tests: riscv_arithmetic_basic_test, iterations:1
> Fri, 08 Nov 2019 21:52:59 INFO     Processing simulator setup file : ../riscv-dv.overlay/yaml/simulator.yaml
...

and I do have the targets:

> simond@lnx6471:~/git/tmp10/riscv-dv$ ls target
> ml  rv32i  rv32imc  rv64gc  rv64imc

but when I  run:

> python3 ./run.py -o ./outdir   --core_setting_dir /home/simond/git/tmp10/riscv-dv/target/rv64gc   --simulator questa32 --simulator_yaml /home/simond/git/tmp10/riscv-dv.overlay/yaml/simulator.yaml   --iss spike,ovpsim --iss_yaml /home/simond/git/tmp10/riscv-dv/yaml/iss.yaml   --seed 1234 --isa rv64gc --mabi lp64   --test riscv_arithmetic_basic_test --testlist /home/simond/git/tmp10/riscv-dv/target/rv64gc/testlist.yaml   --iterations 1   --steps all --verbose 

it works.

any ideas?

Reply:
Please don't use core_setting_dir anymore. I will take a look at rv64gc issue